lms/static/sass/views/_verification.scss

```diff
@@ -821,6 +829,114 @@
 
     // indiv slides - review
     #wrapper-review {
+      color: $black;
+
+      .page-title {
+          border-bottom: 2px solid $darkGrey;
+          padding-bottom: 15px;
+          margin-bottom: 20px;
+          font-weight: bold;
```

Comment:
for font-weights and similar typography related styles, check _typography.scss and use an %extend where possible. In this case, there's a %font-weight rule somewhere in that document to achieve the same goal.